src/components/pages/WelcomePage.tsx

```diff
@@ -238,6 +243,48 @@ class WelcomePage extends React.Component<Props, State> {
     }
   }
 
+  /**
+   * Render Bottom section posts
+   * 
+   */
+  private renderBottomSectionPosts = (categoryId: number) => {
+    const { classes } = this.props;
+    const linkedEventsPost = this.state.linkedEventsPost;
+    if (!this.state.posts) {
+      console.log("NO POSTS");
+      return null;
+    } else {
+      console.log("ALL POSTS ARE ", this.state.posts);
+      return(
+        this.getLimitedPosts(categoryId, 6).map(post => {
+          console.log("BOTTOM SECTION POST IS: ", post.title);
+          return (
+            <div style={{ backgroundImage: `url(${this.getAttachmentForPost(post)})` }} className={classes.bottom_section_item}>
+              <p>{ReactHtmlParser(post.title ? post.title.rendered || "" : "")}</p>
```

Comment:
This should probably work without react html parser? I think post.title.rendered is just a string without any html.